app/views/laboratory_samples/_form_js.haml

```diff
@@ -34,6 +34,10 @@
       e.stopPropagation();
     });
 
+    $('.clear-note').on('click', function(e) {
+      $(this).closest('.note').hide();
```

Comment:
known issue: onclick events are bubbling and `event.stopPropagation` does not seem to fix it.